legacy/src/api/assets/assets.md

```diff
@@ -355,4 +362,4 @@ Archive supported asset types by asset id. Currently only gift cards may be arch
 [Asset Transfers]: {% link api/assets/asset-transfers.md %}
 [paginated]: {% link api/pagination.md %}
 [Settlements]: {% link api/assets/wallets.md %}#settlement-wallets
-[token collection]: {% link api/assets/tokens.md %}#token-collection
```

Comment:
Not sure why there is a change to this line, I didn't change it. As you can see it is replaced with the exact same line.